module/VuFind/src/VuFind/ILS/Driver/AbstractMultiDriver.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+/**
+ * Abstract Multi Driver.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Hebis Verbundzentrale 2023.
```

Comment:
I think you'll need to carry over the copyrights for the existing moved code.

Reply:
Fixed